tensorflow_probability/python/optimizer/bfgs.py

```diff
@@ -78,7 +78,8 @@ def minimize(value_and_gradients_function,
              max_iterations=50,
              parallel_iterations=1,
              stopping_condition=None,
-             name=None):
+             name=None,
+             print_n_iterations=50):
```

Comment:
maybe this should default to None (no printing), and only add the print_op if not None?